web/react/stores/team_store.jsx

```diff
@@ -11,6 +11,12 @@ var BrowserStore = require('../stores/browser_store.jsx');
 
 var CHANGE_EVENT = 'change';
 
+var utils;
+function getWindowLocationOrigin() {
+    if (!utils) utils = require('../utils/utils.jsx');
```

Comment:
Ah, so this threw me for a loop for awhile (still not sure why exactly, maybe you can help there), but every time I tried to do var utils = ... the site would not work due to some random function in user store not being defined.  Once I realized that it was the including of the utils file that was causing it, I started looking at the other stores files to see if utils was defined anywhere else in a different one.  It wasn't _but_ I noticed that in browser_store.jsx we don't define UserStore until it's needed and figured I'd try that approach.  Once that worked I assumed it was something that was needed due to some ordering issue with how we load in the files.  But like you said not a great looking solution so let me know if you can think of something.